src/lib/components/profile/image-input/ImageInput.svelte

```diff
@@ -0,0 +1,63 @@
+<script lang="ts">
+  import deleteIcon from '$lib/icons/delete.svg'
+  import uploadIcon from '$lib/icons/upload.svg'
+  import { slide } from 'svelte/transition'
+
+  export let title: string
+  export let subTitle: string
+  export let handleChange: (event: Event) => void = () => {}
+  export let handleDelete: () => void = () => {}
+  export let isDeletable: boolean = true
```

Comment:
```suggestion
  export let isDeletable = true
```
Creo que no hace falta especificar el tipo